scene/resources/animation.cpp

```diff
@@ -427,6 +427,16 @@ bool Animation::_set(const StringName &p_name, const Variant &p_value) {
 		} else {
 			return false;
 		}
+#ifndef DISABLE_DEPRECATED
+	} else if (prop_name == "loop") { // Compatibility with Godot 3.x.
+		bool loop = p_value;
+		if (loop) {
+			loop_mode = Animation::LoopMode::LOOP_LINEAR;
+		} else {
```

Comment:
One question would be if we should set it when false, or in case it is weirdly set to some non-bool value, some compatibility cases just set it on the `true` case, just minor detail question

Reply:
Could also be done by adding `&& p_value.operator bool()` to the line checking for the property, like is done in some places, as technically we only care about stored values in scene files, and there only the true value is ever stored

Reply:
I think the current code is fine, but yeah this last suggestion may be a tiny bit better so we don't override an existing `loop_mode` value if there happened to be a broken `loop` saved in the file.

Reply:
@Owl-A Would you be able to do this final edit?

Reply:
Sure, I have done it.